edocumentation-service/src/main/java/org/dcsa/edocumentation/service/ShippingInstructionService.java

```diff
@@ -1,38 +1,43 @@
 package org.dcsa.edocumentation.service;
 
 import lombok.RequiredArgsConstructor;
-import org.dcsa.edocumentation.domain.persistence.entity.enums.EblDocumentStatus;
+import org.dcsa.edocumentation.domain.persistence.entity.ShippingInstruction;
+import org.dcsa.edocumentation.domain.persistence.repository.ShipmentEventRepository;
 import org.dcsa.edocumentation.domain.persistence.repository.ShippingInstructionRepository;
-import org.dcsa.edocumentation.domain.persistence.repository.specification.ShippingInstructionSpecification;
 import org.dcsa.edocumentation.service.mapping.ShippingInstructionMapper;
-import org.dcsa.edocumentation.transferobjects.ShippingInstructionSummaryTO;
-import org.dcsa.skernel.infrastructure.pagination.Cursor;
-import org.dcsa.skernel.infrastructure.pagination.PagedResult;
-import org.springframework.lang.Nullable;
+import org.dcsa.edocumentation.transferobjects.ShippingInstructionRefStatusTO;
+import org.dcsa.edocumentation.transferobjects.ShippingInstructionTO;
 import org.springframework.stereotype.Service;
 
-import java.util.Arrays;
+import java.util.Optional;
 
 @Service
 @RequiredArgsConstructor
 public class ShippingInstructionService {
+
+  private final LocationService locationService;
+  private final ShipmentEventRepository shipmentEventRepository;
   private final ShippingInstructionRepository shippingInstructionRepository;
   private final ShippingInstructionMapper shippingInstructionMapper;
 
-  public PagedResult<ShippingInstructionSummaryTO> findShippingInstructionSummaries(
```

Comment:
I take it this was removed because this functionality is now in ShippingInstructionSummaryService. Relates to the comment below